cmd/kubeadm/app/phases/certs/renewal/renewal_test.go

```diff
@@ -214,21 +220,136 @@ func TestRenewExistingCert(t *testing.T) {
 		t.Fatalf("couldn't write out certificate")
 	}
 
+	// makes some time pass
+	time.Sleep(1 * time.Second)
```

Comment:
did you observe flakes if there is no sleep here?
[1]

Reply:
yes, sometimes it flakes without sleep because of Cert.NotAfter is the same in old/new certs if we are too fast

Reply:
We should probably expose some funcs in the pki helpers that allow us to specify `NotBefore` & `NotAfter` for test purposes. But that's a job for another PR.